.github/workflows/main.yml

```
@@ -21,18 +21,18 @@ jobs:
     # Sequence of tasks that will be executed as part of the job
     steps:
     # Checks-out repository under $GITHUB_WORKSPACE, so the job can access it
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)